application.cpp

```diff
@@ -105,7 +105,7 @@ void application::set_program_options()
          ("data-dir,d", bpo::value<std::string>(), "Directory containing program runtime data")
          ("config-dir", bpo::value<std::string>(), "Directory containing configuration files such as config.ini")
          ("config,c", bpo::value<std::string>()->default_value( "config.ini" ), "Configuration file name relative to config-dir")
-         ("logconf,l", bpo::value<std::string>()->default_value( "logging.json" ), "Logging configuration file name/path for library users");
+         ("log-config,l", bpo::value<std::string>()->default_value( "logging.json" ), "Logging configuration file name/path for library users");
```

Comment:
Will renaming this break any scripts (perhaps outside our control)?  Do we care?